### DISCUSSION TITLE
Please set minimal c++ version requirement to c++11

### DISCUSSION THREAD

**Discussion by pewsou** [google/or-tools#2562] [Build and configuration issues] [answered]
It makes hard times to build the SW when c++17 required, but it builds with c++14 and almost entirely with c++11.
Also, for me to build with c++17 under MacOS takes to upgrade the hardware, since support for c++17 requires upgrade to OS X version incompatible with current HW. It seems to be slightly unfair to demand spending on new HW while it evidently can be avoided.

**Comment by Mizux** [comment, marked as answer]
long story short, google move to C++17 as default language dialect and we don't have enough human resource to backport/fix code base to support C++14 or C++11...

What about using homebrew to get a newer version of clang or gcc with a C++17 support ?
